src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -284,4 +291,24 @@ public static bool IsGet(this HttpContext context)
     {
         return string.Equals(context.Request.Method, "get", StringComparison.OrdinalIgnoreCase);
     }
+
+    public static async Task ConfigureRedis(WebApplicationBuilder builder, bool isDevelopment, string? managedIdentityId)
```

Comment:
@premun was reorganizing configuration to be a bit more readable, it might be better if we extracted this in a separate static class

Reply:
Ah actually, this might be used in more than one project. We create `ProductConstructionService.Common` project to put things like this

Reply:
Moved to `ProductConstructionServiceExtension` as it looked like a good fit. Lemme know if its not the best place for it